.zenodo.json

```diff
@@ -0,0 +1,95 @@
+            "name": "Stella, Alessandra"
+        },
+        {
+            "orcid": "0000-0003-0789-6279",
+            "affiliation": "Institute of Neuroscience and Medicine (INM-6), Computational and Systems Neuroscience & Institute for Advanced Simulation (IAS-6), Theoretical Neuroscience, Jülich Research Centre and JARA, Jülich, Germany",
+            "name": "Jurkus, Regimantas"
+        },
+        {
+            "orcid": "0000-0002-9302-5893",
+            "affiliation": "Institute of Neuroscience and Medicine (INM-6), Computational and Systems Neuroscience & Institute for Advanced Simulation (IAS-6), Theoretical Neuroscience, Jülich Research Centre and JARA, Jülich, Germany",
+            "name": "Kramer, Maximilian"
+        },
+        {
+            "affiliation": "Institute of Neuroscience and Medicine (INM-6), Computational and Systems Neuroscience & Institute for Advanced Simulation (IAS-6), Theoretical Neuroscience, Jülich Research Centre and JARA, Jülich, Germany",
+            "name": "Dąbrowska, Paulina"
+        },
+        {
+            "orcid": "0000-0001-7292-1982",
+            "affiliation": "Institute of Neuroscience and Medicine (INM-6), Computational and Systems Neuroscience & Institute for Advanced Simulation (IAS-6), Theoretical Neuroscience, Jülich Research Centre and JARA, Jülich, Germany",
+            "name": "Kern, Moritz"
+        },
+        {
+            "orcid": "0000-0002-9557-1003",
+            "affiliation": "Institute of Neuroscience and Medicine (INM-6), Computational and Systems Neuroscience & Institute for Advanced Simulation (IAS-6), Theoretical Neuroscience, Jülich Research Centre and JARA, Jülich, Germany",
+            "name": "Kurth, Anno Christopher"
+        },
+        {
+            "orcid": "0000-0001-7373-5962",
+            "affiliation": "Institute of Neuroscience and Medicine (INM-6), Computational and Systems Neuroscience & Institute for Advanced Simulation (IAS-6), Theoretical Neuroscience, Jülich Research Centre and JARA, Jülich, Germany",
+            "name": "Gutzen, Robin"
+        },
+        {
+            "orcid": "0000-0003-2401-7862",
+            "affiliation": "Cognitronics and Sensor Systems, CITEC, Bielefeld University, Bielefeld, Germany",
+            "name": "Porrmann, Florian"
+        },
+        {
+            "orcid": "0000-0002-3168-7365",
+            "affiliation": "Cognitronics and Sensor Systems, CITEC, Bielefeld University, Bielefeld, Germany",
+            "name": "Pilz, Sarah"
+        }
+    ],
+
+    "keywords": [
+    "neuroscience",
+    "neurophysiology",
+    "statistics",
+    "data-analysis"
+    ],
+
+    "license": {
+      "id": "BSD-3-Clause"
+    },
+
+    "related_identifiers": [
+        {
+            "scheme": "doi",
+            "identifier": "10.12751/incf.ni2018.0019",
+            "relation": "isDocumentedBy",
+            "resource_type": "publication-article"
+        }
+    ],
+
+    "upload_type": "software"
+}
```

Comment:
Is there a way to specify acknowledgements/Funding in this json format?

Reply:
With commit https://github.com/NeuralEnsemble/elephant/pull/459/commits/1a7108d1e814b06e2897e04f698b51ca2491948e , ids No. 720270, No. 785907 and No. 945539 (Human Brain Project SGA1, SGA2 and SGA3) are added.